src/Text/Pandoc/Shared.hs

```diff
@@ -47,6 +47,7 @@ module Text.Pandoc.Shared (
                      compactify,
                      compactifyDL,
                      linesToPara,
+                     figureDiv,
```

Comment:
This definitely needs to be announced as an API change in the commit message.
(Worth also checking the rest of the changes to make sure there are no other API changes.)

Reply:
Added a bullet point to the commit message. I checked again and didn't find any other API changes.